cmd/agate/main.go

```diff
@@ -95,6 +97,33 @@ func initialModel(subprocess string) model {
 	debugLogger := debug.InitDebugLogger()
 	debug.DebugLog("Debug logger initialized successfully")
 
+	// Load user configuration and apply keybinding overrides
+	configFile, err := config.GetAgateDir()
+	if err == nil {
+		configFile = filepath.Join(configFile, "config.yaml")
```

Comment:
nitpick but wondering if we should have a dedicated method for grabbing the `config.yaml` - feels like we will be calling it a lot as it grows